ale_linters/qml/qmllint.vim

```diff
@@ -0,0 +1,30 @@
+" Author: pylipp
+" Description: qmllint for QML files
+
+" Find lines like
+" /home/foo_user42/code-base/qml/Screen.qml:11 : Expected token `}'
+function! ale_linters#qml#qmllint#Handle(buffer, lines) abort
+    let l:pattern = '\v^[/_-a-zA-z0-9\. ]+:(\d+) : (.*)$'
+    let l:output = []
+
+    for l:match in ale#util#GetMatches(a:lines, l:pattern)
+        let l:item = {
+        \   'lnum': l:match[1] + 0,
+        \   'col': 0,
+        \   'text': l:match[2],
+        \   'code': '',
+        \   'type': 'E',
+        \}
+        call add(l:output, l:item)
+    endfor
+
+    return l:output
+endfunction
+
+call ale#linter#Define('qml', {
+\   'name': 'qmllint',
+\   'output_stream': 'stderr',
+\   'executable': 'qmllint',
+\   'command': 'qmllint %s',
```

Comment:
`%s` will be replaced with the filename. Could you use `%t` here to use a temporary file instead? You can't check the file itself while you type. If checking a temporary file doesn't work, you should add `'lint_file': 1`, on the end, so ALE will only check for problems when you open a file or save it, etc. Then you'd need to edit the documentation to contain a `!!` marker next to the linter, so people know it can't be used for linting while you type.